app/api/scrypt.native.ts

```diff
@@ -0,0 +1,21 @@
+import { ScryptProvider, Scrypt } from '@defichain/jellyfish-wallet-encrypted'
+import * as crypto from 'expo-crypto'
+
+// TODO: this is quick mock, required secured secret derivation
+class Mock implements ScryptProvider {
```

Comment:
As how it is named, this is not finalized. Major work/research require to make crypto safe randombytes and pbkdf2 works in mobile platforms.

NOTE: browserify randombytes do not work for mobile.